pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -16,16 +16,25 @@ import (
 )
 
 const (
-	keyPattern   = "%s_%s"
-	invKeyPrefix = "inv_"
+	keyPattern        = "%s_%s"
+	invKeyPrefix      = "inv"
+	connIDKeyPrefix   = "conn"
+	protocolKeyPrefix = "pro"
 )
 
 // ConnectionRecord contain info about did exchange connection
 type ConnectionRecord struct {
```

Comment:
Just wondering if we can rename this to Connection. Record doesn't add much context imo. That way our APIs will be more cleaner, ie. GetConnection returns Connection.

Reply:
// Connection connection
type Connection struct {
	DID    string   `json:"did,omitempty"`
	DIDDoc *did.Doc `json:"did_doc,omitempty"`
}
Connection is already in the models which is used in the request and response